winrt/docsrc/CanvasFontFace.xml

```diff
@@ -130,6 +130,12 @@ Licensed under the MIT License. See LICENSE.txt in the project root for license
     <member name="M:Microsoft.Graphics.Canvas.Text.CanvasFontFace.GetGlyphIndices(System.UInt32[])">
       <summary>Returns the nominal mapping of UCS4 Unicode code points to glyph indices as defined by the font 'CMAP' table.</summary>
     </member>
+    <member name="M:Microsoft.Graphics.Canvas.Text.CanvasFontFace.GetGlyphMetrics(System.Int32[],System.Boolean)">
+      <summary>Gets the metrics and bounds of the glyphs that would get drawn in em units.</summary>
+    </member>
+    <member name="M:Microsoft.Graphics.Canvas.Text.CanvasFontFace.GetGdiCompatibleGlyphMetrics(System.Single,System.Single,System.Numerics.Matrix3x2,System.Boolean,System.Int32[],System.Boolean)">
+      <summary>Gets the metrics and bounds of the glyphs that would get drawn, compatible with what GDI would produce, in pixel units.</summary>
+    </member>
```

Comment:
True- but it isn't right to call these 'pixel units'- if anything, it would be DIPs.

Now that I'm seeing it, could you please make GetGdiCompatibleGlyphMetrics not scale by fontSize. It uses fontSize to compute its result, but it should have consistent semantics with GetGlyphMetrics, which returns units in em space. It's up to the app to scale by the font size; as it may also apply a DPI transform or some scaling from the passed-in transform.